### PR TITLE
Finish adding in the manual bindings.

### PR DESCRIPTION
**Pull request by creationix**
- Add type definitions for `egl` and `ssl` modules.
- Add comments to `png` and `jpeg` modules.
- Cleanup comments on `gl` module.
- Add `uv`, `gl`, `png`, `jpeg`, `ssl`, and `egl` modules to generated API docs.
- Add `egl` and `ssl` to npm package for editor autocomplete.
- Bump version of npm package to `magic-script-typings@1.6.3`.
- Use `longTitle` in typedoc to greatly improve visual context of the many modules.
- Remove `max-width` constraint on main content to improve experience on wide monitors.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
   "description": "Typescript definitions for MagicScript",
   "files": [
+    "egl",
     "gl",
     "jpeg",
     "lumin",
     "mxs",
     "png",
-    "uv",
-    "index.d.ts",
```
Comment (kpiascik) [inline package.json]: do we want to remove this file from the repository if we're not using it?

Comment (creationix) [reply package.json]: I thought it might be used by typedoc.  Let me verify it's not used by typedoc or vscode.

Comment (creationix) [reply package.json]: Yep, just verified.  It's not used anywhere.  Removing...

File: package.json [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-    "index.js"
+    "ssl",
+    "uv"
   ],
   "scripts": {
     "build": "node index.js",
-    "typedoc": "typedoc ./lumin",
+    "typedoc": "typedoc ./png ./jpeg ./uv ./gl ./egl ./ssl ./lumin",
```
Comment (kpiascik) [inline package.json]: we're missing `mxs`

Comment (kpiascik) [reply package.json]: please use the same list and order as above in `files`

Comment (creationix) [reply package.json]: Done. Thanks.

**Pull request merged**